app/views/inventories/index.html.erb

```diff
@@ -1 +1,23 @@
-<h1> This page would be for all the listed inventories by a given user </h1>
+<div class="container mt-3">
+  <h1 class="text-center mb-3">Your Inventories</h1>
+  <% if current_user %>
+    <%= link_to "New Inventory", new_inventory_path, class: "btn btn-primary" %>
```

Comment:
[OPTIONAL] We can move this to the right side to maintain the style of the website.